pages.ja/common/aws-configure.md

```diff
@@ -0,0 +1,36 @@
+# aws configure
+
+> AWS CLI の設定を管理します。
+> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>
```

Comment:
```suggestion
> 詳しくはこちら: <https://docs.aws.amazon.com/cli/latest/reference/configure/>.
```

Reply:
@kbdharun 
Thanks for the review!
It looks good to me, so I'll adopt it.